packages/block-editor/src/components/list-view/use-list-view-client-ids.js

```diff
@@ -76,13 +57,32 @@ export default function useListViewClientIds(
 	showOnlyCurrentHierarchy,
 	__experimentalPersistentListViewFeatures
 ) {
-	const selectedClientIds = useListViewSelectedClientIds(
-		__experimentalPersistentListViewFeatures
+	const { selectedClientIds, draggedClientIds } = useSelect(
+		( select ) => {
+			const {
+				getSelectedBlockClientId,
+				getSelectedBlockClientIds,
+				getDraggedBlockClientIds,
+			} = select( blockEditorStore );
+
+			if ( __experimentalPersistentListViewFeatures ) {
+				return {
+					selectedClientIds: getSelectedBlockClientIds(),
+					draggedClientIds: getDraggedBlockClientIds(),
+				};
+			}
+
+			return {
+				selectedClientIds: getSelectedBlockClientId(),
```

Comment:
For other reviewers getSelectedBlockClient**Ids** vs getSelectedBlockClientIds**Id**